readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://<adrian200002>.github.io/layout_stars/)
+- [TEST REPORT LINK](https://<adrian200002>.github.io/layout_stars/report/html_report/)
```

Comment:
Please pay attention to those links... (don't put `<>` inside of them)
And also attach them to PR description
```suggestion
- [DEMO LINK](https://adrian200002.github.io/layout_stars/)
- [TEST REPORT LINK](https://adrian200002.github.io/layout_stars/report/html_report/)
```